docs/README.md

```diff
@@ -1,12 +1,10 @@
 # nf-core/methylseq: Documentation
 
-The nf-core/methylseq documentation is split into the following files:
+The nf-core/methylseq documentation is split into the following pages:
 
-1. [Installation](https://nf-co.re/usage/installation)
-2. Pipeline configuration
-    * [Local installation](https://nf-co.re/usage/local_installation)
-    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
-    * [Reference genomes](https://nf-co.re/usage/reference_genomes)
-3. [Running the pipeline](usage.md)
-4. [Output and how to interpret the results](output.md)
-5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
+* [Usage](usage.md)
```

Comment:
```suggestion
* [Usage](https://nf-co.re/methylseq/usage)
```

Reply:
I think that this will break the nf-core linting, as it came from the template. The relative URL should work on the website as well, right?

Reply:
ah, okay, sorry I got confused, this is `docs/readme` so it will actually not be rendered on the website. So it can stay how it is.